### PR TITLE
Protect: polish interstitial page in medium viewport size

### PR DESCRIPTION
**Pull request by retrofox**
This PR reduces the padding in both sections of the Interstitial page when on medium viewport size.
Also, it sets up the Dialog layout depending on this breakpoint, from `5 | 3` to `4 | 4`

Disclaimer: there is still an issue that we need to tackle to make the UI better in different screen sizes.

Fixes #

#### Changes proposed in this Pull Request:

* Set up the Dialog grid depending on the viewport size.
* Tweak primary and secondary sections of Dialog in small and medium viewport sizes

#### Testing instructions:

* Go to Protect dashboard
* Disconnect the site in case it's already connected
* Take a look at the interstitial page
* play with the viewport size. You'd like to use the mobile tool of our client
* Confirm the page looks better

before  | after
------|-------
image | image
image | image

### CODE REVIEW

File: projects/plugins/protect/src/js/components/interstitial/index.jsx [hunk 1]
```diff
@@ -58,10 +59,19 @@ const SecurityBundle = ( { onAdd, redirecting, rest } ) => {
  * @returns {React.Component} Interstitial react component.
  */
 const Interstitial = ( { onSecurityAdd, securityJustAdded } ) => {
+	const [ isLowerThanLarge ] = useBreakpointMatch( 'lg', '<=' );
```
Comment (renatoagds) [inline projects/plugins/protect/src/js/components/interstitial/index.jsx:62]: This is right? You're using `<=`, which means everything, and not lower than large.

Comment (retrofox) [reply projects/plugins/protect/src/js/components/interstitial/index.jsx:62]: it's WIP PR.

Comment (retrofox) [reply projects/plugins/protect/src/js/components/interstitial/index.jsx:62]: I've removed the second arg in the hook.

**Review by renatoagds — APPROVED** [review]
LGTM

**Pull request merged**